charts/unified-chart/templates/ingress/alb-ingress.yaml

```diff
@@ -5,7 +5,7 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ required ".Values.name is required" .Values.name }}-alb-ingress
+  name: {{ required ".Values.name is required" .Values.name }}-alb
```

Comment:
adding `alb` suffix is great but one might assume that the other Ingress isn't alb. I suggest, Blue and Green